python-project/test/test_jp.py

```diff
@@ -45,6 +45,11 @@ def png_dir(test_case: OCRTest) -> Path:
 
     return Path(data, 'jp', name, 'png')
 
+def txt_dir(test_case: OCRTest) -> Path:
```

Comment:
Since this is exactly the same as the `png_dir` function except for the file extension, could you generalize them to avoid duplicating the logic?